src/Sentry/Internal/Extensions/JsonExtensions.cs

```diff
@@ -281,9 +276,9 @@ public static void Deconstruct(this JsonProperty jsonProperty, out string name,
             {
                 writer.WriteStringValue(formattable.ToString(null, CultureInfo.InvariantCulture));
             }
-            else if (value is Type t)
```

Comment:
Can we move this entire if/else sequence inside `SentryJsonConverter`?

Reply:
Removed since SentryJsonConverter already did that. Previously it was failing before entering the converter but turned out after the last changes, the problem isn't occurring.